source/documentation/other-topics/persistent-volume-encryption-snapshot.html.md.erb

```diff
@@ -0,0 +1,128 @@
+---
+title: Persistent volume encryption at rest and daily snapshots
+last_reviewed_on: 2019-12-24
+review_in: 3 months
+---
+
+# <%= current_page.data.title %>
+
+
+### Introduction
+
+This guide will give you an overview of how to set encryption at rest for the Persistent volumes and daily snapshots taken to back up Persistent volumes. 
+
+
+### Encrypting Persistent volumes
+
+For users who need encryption at rest enabled for persistence volumes, Cloud Platform team created [gp2-expand][gp2-expand] storge class. this `gp2-expand` [Storage Class][storge class] have set with parameter `encrypted: "true"`, which denotes the EBS volume should be encrypted.
+
+The name of a StorageClass object is significant, user should match the PersistentVolumeClaim to [gp2-expand][gp2-expand] Storage class, by setting the storageClassName attribute to the name of a `gp2-expand`, as shown in the example below. 
+
+
+```
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: myclaim
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 8Gi
+  storageClassName: gp2-expand
+  selector:
+    matchLabels:
+      release: "stable"
+    matchExpressions:
+      - {key: environment, operator: In, values: [dev]}
+
+```
+
+
+Using the above PersistentVolumeClaim example, the cluster will try to [dynamically provision][dynamic-provisioning] a [Persistent volume][persistent-volumes] specially for the PersistentVolumeClaim. This provisioning is based on [gp2-expand][gp2-expand] [Storage Class][storage-class-aws-ebs], which will encrypt the PersistentVolume using the [KMS][amazon-kms].
+
+
+### Snapshots to back up Persistent volumes
+
+Users may want to have snapshots taken to back up Persistent volumes, that can be done using the [snapshot lifecycle policy][aws_dlm_lifecycle_policy]. This example snapshot lifecycle policy below will create a snapshot of all tagged volumes, every 24 hours starting at 05:00 UTC. A maximum of 30 snapshots will be retained of a target volume. The oldest snapshot retained will be <= 1 months old. 
+
+Using the example below create a `dlm.tf` file in [environments repo][env-repo] under your namespace/resource. Ammend the `tags_to_add` and replace `<pvc-name>` with the name of your PersistentVolumeClaim, you can get the name of your PersistentVolumeClaim, by running below command.
```

Comment:
We should explain that "dlm" stands for "data lifecycle management", the first time we use the term.